### PR TITLE
Bump Groovy from 2.4.12 to 2.4.21

### PR DESCRIPTION
**Pull request by basil**
Not for review. Filing for testing purposes. Timestamped snapshots built locally (with passing tests!) available at:

https://repo.jenkins-ci.org/snapshots/org/kohsuke/groovy-sandbox/1.28-SNAPSHOT/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>org.codehaus.groovy</groupId>
       <artifactId>groovy</artifactId>
-      <version>2.4.12</version> <!-- Note: Do not update past 2.4.12 unless Jenkins core is updated to use a newer version of Groovy. -->
+      <version>2.4.21</version> <!-- Note: Do not update past 2.4.12 unless Jenkins core is updated to use a newer version of Groovy. -->
```
Comment (jglick) [inline pom.xml:112]:
```suggestion
      <version>2.4.21</version> <!-- Note: Do not update past this version unless Jenkins core is updated to use a newer version of Groovy. -->
```

**Comment by jglick** [comment]
Eligible for merge now?

**Comment by basil** [comment]
> Eligible for merge now?

Yes I think so.

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
also #70, #71

**Review by car-roll — APPROVED** [review]

**Pull request merged**